src/Runner/Runner.php

```diff
@@ -95,6 +101,27 @@ public function run()
 		}
 		$this->jobCount = count($this->jobs) + array_sum($this->results);
 
+		if ($this->tempDir) {
+			$this->orderFile = $this->tempDir . DIRECTORY_SEPARATOR . 'Tester.Runner-' . substr(md5(__FILE__), 0, 5) . '.list';
+			touch($this->orderFile);
```

Comment:
Probably yes. Will see how it will work in practice. Can by changed later to global, not hashing at all.